Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -9340,6 +9342,9 @@
 		44AE045A2579CFFB00C42E65 /* AppHighlightRangeData.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = AppHighlightRangeData.h; sourceTree = "<group>"; };
 		44B38BF42536901A00A4458D /* SelectionRangeData.h */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.h; path = SelectionRangeData.h; sourceTree = "<group>"; };
 		44B38BF925369A8800A4458D /* SelectionRangeData.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = SelectionRangeData.cpp; sourceTree = "<group>"; };
+		44BC1B2528A1DAE300C7DF2D /* Header.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; name = Header.h; path = ../../../Header.h; sourceTree = "<group>"; };
```

Comment:
This seems suspicious.  Could you remove all references to Header.h?  I don't think it's actually added.